lib/asciidoctor/stylesheets.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# Monkey patch the whole file because the class is static
```

Comment:
Keep in mind it has to be an extra script not at the same path as the original file. In other words, it needs to be appended to the built-in script. So put it at the path lib/asciidoctor/ext/stylesheets.rb or something.